### PR TITLE
T cases

### PR DESCRIPTION
**Pull request by TechWriter81**
add test-cases results

### CODE REVIEW

File: test-cases/aadr.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+## Steps
+
+
+### Test Step № 1
+
+Go to aadr
+
+```bash
+c0d /media/data/aadr
```
Comment (alex-medvedev-msc) [inline test-cases/aadr.md:28]: c0d should be cd?

File: test-cases/ibis.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+#### Test result
+
+After executing the command, the file  `metrics.tsv` containing metrics will appear:
+
+| True Degree | Precision | Recall |
+|:--|:--|:--|
+| 1  | 1.0  | 1.0  |
+| 2  | 1.0  | 1.0  |
+| 3  | 0.9944444444444445  | 0.9944444444444445  |
```
Comment (alex-medvedev-msc) [inline test-cases/ibis.md:38]: May be we should round up precision and recall to 3-4 significant digits?

**Review by alex-medvedev-msc — CHANGES_REQUESTED** [review]

**Review by josephkott — CHANGES_REQUESTED** [review]
For `aadr.md` and `khazar.md`, I would suggest to remove sections "Test result", "Status", "Notes" from all steps except for the last one, since only the last step contains the actual test result.

`ibis.md` and `king.md` are ok.

**Review by alex-medvedev-msc — APPROVED** [review]

**Comment by TechWriter81** [comment]
comments are worked out

**Pull request merged**